README.md

```diff
@@ -123,23 +158,40 @@ node_modules/uswds/
     └── stylesheets/
 ```
 
-`require('uswds')` will load all of the Draft U.S. Web Design Standards’ JavaScript onto the page. The `uswds` module itself does not export anything.
+#### Importing assets
+
+Since you are already using `npm`, the Draft U.S. Web Design Standards team recommends leveraging the ability to write custom scripts. Here are some links to how we do this with our docs website using `npm` + [`gulp`](http://gulpjs.com/):
+
+[Link to `npm` scripts example in `web-design-standards-docs`](https://github.com/18F/web-design-standards-docs/blob/staging/package.json#L4)
+[Link to gulpfile.js example in `web-design-standards-docs`](https://github.com/18F/web-design-standards-docs/blob/staging/gulpfile.js)
 
-The main Sass (SCSS) source file is here:
+#### Sass
+
+The Standards are easily customizable using the power of [Sass (Syntactically Awesome Style Sheets)](http://sass-lang.com/). The main Sass (SCSS) source file is located here:
 
 ```
-node_modules/uswds/src/stylesheets/all.scss
+node_modules/uswds/src/stylesheets/uswds.scss
 ```
 
-The non-minified CSS that’s been precompiled is here:
+Global variables are defined in the `node_modules/uswds/src/stylesheets/core/_variables.scss` file. Custom theming can be done by copying the `_variables.scss` file into your own project’s Sass folder, changing applicable variable values, and importing it before `uswds.scss`. 
+
+Below is an example of how you might setup your main Sass file to achieve this:
 
 ```
-node_modules/uswds/dist/css/uswds.css
+@import 'variables.scss' # Custom Sass variables file
+@import 'node_modules/uswds/src/stylesheets/uswds.scss';
```

Comment:
@shawnbot I thought about that too. Based on our user type, I'd rather be more explicit for users who may not understand why it's 👌 to not have the extension here. 

Cool?